format/Layout.md

```diff
@@ -384,37 +384,42 @@ The layout for [{'joe', 1}, {null, 2}, null, {'mark', 4}] would be:
 
 * Children arrays:
   * field-0 array (`List<char>`):
-    * Length: 4, Null count: 1
+    * Length: 4, Null count: 2
     * Null bitmap buffer:
 
       | Byte 0 (validity bitmap) | Bytes 1-63            |
       |--------------------------|-----------------------|
-      | 00001101                 | 0 (padding)           |
+      | 00001001                 | 0 (padding)           |
 
     * Offsets buffer:
 
       | Bytes 0-19     |
       |----------------|
-      | 0, 3, 3, 6, 10 |
+      | 0, 3, 3, 3, 7  |
 
      * Values array:
-        * Length: 10, Null count: 0
+        * Length: 7, Null count: 0
         * Null bitmap buffer: Not required
 
         * Value buffer:
 
-          | Bytes 0-9      |
+          | Bytes 0-6      |
           |----------------|
-          | joebobmark     |
+          | joemark        |
 
   * field-1 array (int32 array):
-    * Length: 4, Null count: 0
-    * Null bitmap buffer: Not required
+    * Length: 4, Null count: 1
+    * Null bitmap buffer:
```

Comment:
I think this is OK, but strictly speaking it would be valid for the null count to be 0, and for the 3rd value slot to be some value. So what was here before was not incorrect (except for the missing padding bytes)